package.json

```diff
@@ -28,13 +28,18 @@
   "dependencies": {
     "aws-auto-auth": "^0.1.1",
     "aws-sdk": "^2.1.31",
+    "centroid-stream": "^0.3.0",
+    "commander": "^2.8.1",
     "csv-to-vrt": "0.0.2",
+    "end-of-stream": "https://registry.npmjs.org/end-of-stream/-/end-of-stream-1.1.0.tgz",
```

Comment:
Good question ;) I'll fix it. I think it had something to due with installing it with a shrinkwrap file already in place.